instrumentation/opentelemetry-instrumentation-pika/tests/test_pika_instrumentation.py

```diff
@@ -72,8 +73,11 @@ def test_instrument_consumers(
         self, decorate_callback: mock.MagicMock
     ) -> None:
         tracer = mock.MagicMock(spec=Tracer)
+        callback_attr = PikaInstrumentor._consumer_callback_attribute_name()
```

Comment:
add a test for `_consumer_callback_attribute_name`

Reply:
@oxeye-nikolay What should it test? If it returns the wrong value, other tests will definitely fail.

Reply:
mock the return value of `pkg_resources.get_distribution("pika").version` and ensure the value returned is good. The tests may fail also without it, but you want the tests to identify the source

Reply:
@oxeye-nikolay I'm longer using `pkg_resources` but rather accessing `pika.__version__` directly, as suggested by owais. If I override the value exported by the package directly it might have unexpected implications and I'd rather avoid it

Reply:
Here, try this:
```python
import pika
from unittest import mock
print(pika.__version__)
with mock.patch("pika.__version__", "0.1.1"):
     print(pika.__version__)
print(pika.__version__)
```
the output should be:
```
1.2.0
0.1.1
1.2.0
```